packages/components/src/core/styles/common/defaultTheme.ts

```diff
@@ -239,40 +296,52 @@ export const defaultAppTheme: AppTheme = {
   },
 };
 
+const lightAppTheme: AppTheme = {
+  colors: lightThemeColors,
+  mode: "light",
+  ...sharedAppTheme,
+};
+
+const darkAppTheme: AppTheme = {
+  colors: darkThemeColors,
+  mode: "dark",
+  ...sharedAppTheme,
+};
+
 // (mlila) whenever our theme uses colors, we need to make sure we allow consuming
 // applications to override those colors using their own custom theme.
 // By defining borders using defaultAppTheme.colors instead of defaultThemeColors,
 // we allow other apps to specify their colors once, and have them apply
-// throughtout the application, such as in borders, etc without having to manually
+// throughout the application, such as in borders, etc without having to manually
 // override every theme property that makes use of colors.
-defaultAppTheme.borders = {
+lightAppTheme.borders = {
```

Comment:
I didn't do this for `darkAppTheme` yet because I'm not sure the best approach here. Just a lot of copy/pasting. Might want some simple functions to generate the objects based on the theme passed in?

Reply:
Done!